stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -133,19 +137,34 @@ def step(self, actions):
         self.step_async(actions)
         return self.step_wait()
 
-    def get_images(self):
+    def get_images(self, *args, **kwargs) -> List[np.ndarray]:
```

Comment:
```suggestion
    def get_images(self, *args, **kwargs) -> Iterable[np.ndarray]:
```
I don't see any random access, but if there is then maybe `Sequence`?